libs/core/src/components/pds-text/pds-text.scss

```diff
@@ -0,0 +1,187 @@
+:host {
+  display: block;
+}
+
+// TODO: Update all values to match new semantic tokens, if applicable
```

Comment:
Is this TODO handled? If not, is it possible to align it with a specific ticket to ensure it's addressed?

Reply:
Just created a ticket to address this.